docs/knowledge/searching-on-github.md

```
@@ -0,0 +1,147 @@
+
+You can search for issues and pull requests on GitHub and narrow the results using these search qualifiers in any combination.
+
+You can search for issues and pull requests globally across all of GitHub, or search for issues and pull requests within a particular organization. For more information, see [About searching on GitHub](/en/search-github/getting-started-with-searching-on-github/about-searching-on-github).
+
+> \[!TIP]
+>
+> * This article contains example searches on GitHub.com, but you can use the same search filters on other GitHub platforms.
+> * You can build advanced filters using boolean and nested queries on your repository's issues page and the issues dashboard. See [Filtering and searching issues and pull requests](/en/issues/tracking-your-work-with-issues/using-issues/filtering-and-searching-issues-and-pull-requests#building-advanced-filters-for-issues).
+> * For a list of search syntaxes that you can add to any search qualifier to further improve your results, see [Understanding the search syntax](/en/search-github/getting-started-with-searching-on-github/understanding-the-search-syntax).
+> * Use quotations around multi-word search terms. For example, if you want to search for issues with the label "In progress," you'd search for `label:"in progress"`. Search is not case sensitive.
+> * Use a minus (hyphen) symbol to exclude results that match a qualifier. For example, to ignore issues created by the "octocat" user, you'd use `-author:octocat` in your search. Note that this does not work for [missing metadata qualifiers](#search-by-missing-metadata).
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)